packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -14,15 +14,21 @@ import root from 'window-or-global';
  * @type {number}
  * @private
  */
-const _partnerId = process.env.KALTURA_PARTNER_ID || 1773841;
+const _partnerId =
+  process.env.REACT_APP_KALTURA_PARTNER_ID ||
```

Comment:
@NSDrowned can you add in a check if `process` exists? This would cause errors if not running on Node.

Reply:
You're right, just added the check.